add center of mass and bounding box codes

vkt1414 opened the pull request:
The config map now includes these two general pyradiomics features.

vkt1414 committed:
add center of mass and bounding box codes

fedorov commented:
Is encoding in SR going to be part of the same PR Vamsi?

When encoding in SR, those two new items will need to be encoded as points, which you have not done before - we will need to look together.

vkt1414 commented:
> Is encoding in SR going to be part of the same PR Vamsi?
> 
Yes, we can use this PR to include both addition of codes, and encoding them in SR. I was just about to test creating the SR for general features. 
> When encoding in SR, those two new items will need to be encoded as points, which you have not done before - we will need to look together.
Does that mean we can not use the standard TID1500?

vkt1414 commented:
> Does that mean we can not use the standard TID1500?

I realize we cannot use value, quantity, and units as bounding box, and the center of mass are qualitative features I guess. We can discuss this tomorrow.

vkt1414 committed:
distinct colors to better visualize vertebra

vkt1414 committed:
modify colors for better visualization
While verifying the vertebrae segmentions, we found that
having different colors would have been very helpful to
identify overlap of vertebrae and the order. Deepa,
subsquently worked on this and curated these colors.

vkt1414 committed:
Created using Colaboratory

vkt1414 committed:
distinct colors used for vertebrae, ribs etc for better visualization

vkt1414 committed:
fix and match colors from thorasic vertebrae to ribs

vkt1414 committed:
match vertebrae and rib colors in the json and yaml files as well
